src/Modal.soy

```diff
@@ -17,7 +18,7 @@
 		class="modal{$elementClasses ? ' ' + $elementClasses : ''}"
 		style="display: {$visible ? 'block' : 'none'}">
 		<div
-			class="modal-dialog"
+			class="modal-dialog{$dialogClasses ? ' ' + $dialogClasses : ''}"
```

Comment:
Hey @fernandosouza, sadly soy doesn't have any default value functionality. The only alternative I've come across is creating a new temp variable that does the same check we are currently doing inline. Something like this:

```
{let $dialogClasses_: $dialogClasses ? ' ' + $dialogClasses : '' /}
```

In this case I don't think it would be worth the extra lines of code.